src/lwt_chain.erl

```diff
@@ -8,22 +14,30 @@
 
 -export([start_link/1]).
 
--export([add_hotspot/2, fund_owner/2]).
+-export([add_hotspot/2]).
+
+-export([height/0, state/0]).
 
 -record(state, {
     oracles = [],
     %% account => balance
     dc_balances = #{},
     %% address => owner
     hotspots = #{},
+    %% val_address => {owner, init_height}
```

Comment:
is overstaking going to be a thing here? should you record the stake in the metadata? (also shouldn't this data structure be defined as a record?)

Reply:
> is overstaking going to be a thing here?
Uh, maybe. Don't really wanna go there in first pass

> should you record the stake in the metadata? (also shouldn't this data structure be defined as a record?)
We could yeah, but more interesting/relevant piece is that LWT should get debited/locked out of owner's account which _is_ tracked in `lwt` contract's state.